src/components/login/container/index.jsx

```diff
@@ -42,54 +42,72 @@ export class Login extends Component {
   render() {
     const { username, password } = this.state;
     const { loading } = this.props;
-    const buttonTitle = loading ? 'Signing in...' : 'Sign in';
+    const buttonTitle = loading ? 'Signing in...' : 'Log in';
```

Comment:
Since you changed 'Sign in' to 'Log in', you should also change 'Signing in...' to 'Logging in' to make it consistent.

Reply:
Thanks for that observation